File: MODUL03/PraktikumPy/PRAK302-231081711004-AbdurrahmanGilangHarjuna.py
nilai = int(input("Masukkan nilai: "))

if nilai >= 90:
    hasil = 'A'
elif nilai >= 80:
    hasil = 'B'
elif nilai >= 70:
    hasil = 'C'
elif nilai >= 60:
    hasil = 'D'
else:
    hasil = 'E'

print(f"Nilai {nilai}")
print(f"Grade anda {hasil}")

File: MODUL03/PraktikumC/PRAK302-231081711004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>

void main(){
    int nilai;
    char grade;

    printf("Program Grade Nilai\n");
    printf("Masukkan nilai: ");
    scanf("%d", &nilai);

    if (nilai >= 80) {
        grade = 'A';
    } else if (nilai >= 70 && nilai <= 79) {
        grade = 'B';
    } else if (nilai >= 60 && nilai <= 69) {
        grade = 'C';
    } else if (nilai >= 50 && nilai <= 59) {
        grade = 'D';
    } else {
        grade = 'E';
    }

    printf("Nilai: %d\n", nilai);
    printf("Grade anda: %c\n", grade);
}